src/cpp/tests/automation/testthat/test-automation-completions.R

```diff
@@ -28,3 +28,26 @@ test_that("autocompletion doesn't trigger active bindings", {
    expect_false(tail(output, 1) == "[1] \"active\"")
    
 })
+
+# https://github.com/rstudio/rstudio/issues/13611
+test_that("autocompletions within piped expressions work at start of document", {
+   
+   code <- .rs.heredoc('
+      
+      mtcars |> mutate(x = mean())
+   ')
+   
+   remote$documentOpen(".R", code)
+   editor <- remote$editorGetInstance()
+   editor$gotoLine(2L, 26L)
+   remote$keyboardExecute("<Tab>")
+   popupEl <- remote$jsObjectViaSelector("#rstudio_popup_completions")
+   popupText <- popupEl$innerText
+   expect_true(grepl("trim =",  popupText, fixed = TRUE))
+   expect_true(grepl("na.rm =", popupText, fixed = TRUE))
+   expect_true(grepl("... =",   popupText, fixed = TRUE))
```

Comment:
Hi @kevinushey! Thanks for taking a look at this issue! However I am not sure this is the correct fix

```suggestion
   expect_true(grepl("x =",  popupText, fixed = TRUE))
   expect_true(grepl("... =",   popupText, fixed = TRUE))
```

And I would expect this to be true if not preceded by a single blank line.

See my comment https://github.com/rstudio/rstudio/issues/13611#issuecomment-1711857799

Reply:
See also https://github.com/davidhodge931/ggblanket/issues/661 im case that's clearer